SMBLibrary/NTFileStore/Structures/FileInformation/Query/FileAttributeTagInformation.cs

```diff
@@ -0,0 +1,55 @@
+﻿/* Copyright (C) 2017 Tal Aloni <tal.aloni.il@gmail.com>. All rights reserved.
+ * 
+ * You can redistribute this program and/or modify it under the terms of
+ * the GNU Lesser Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ */
+using System;
+using System.Collections.Generic;
+using Utilities;
+
+namespace SMBLibrary
+{
+    /// <summary>
+    /// [MS-FSCC] 2.4.14 - FileAttributeTagInformation

```

Comment:
isn't it 2.4.6? ( https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-fscc/d295752f-ce89-4b98-8553-266d37c84f0e )